src/main/java/com/mathworks/ci/BuildArtifactAction.java

```diff
@@ -40,6 +42,9 @@ public BuildArtifactAction(Run<?, ?> build) {
         }
     }
 
+    public String getActionID(){
+        return this.actionID;
+    }
```

Comment:
New line after this for consistent white space?

Reply:
Done.